README.rst

```diff
@@ -55,7 +55,7 @@ Run
 Links
 -----
 
-* `Read the full documentation here <http://pydocstyle.org>`_.
+* `Read the full documentation here <http://www.pydocstyle.org/en/5.0.2/>`_.
```

Comment:
You shouldn't hard-code it. Instead use http://pydocstyle.org/en/stable/

Reply:
Done, Thankx.